web/themes/custom/cloudy/templates/field/field--field-start-time.html.twig

```diff
@@ -1,6 +1,20 @@
 {% set icon %}
   <i class="fas fa-clock"></i>
 {% endset %}
-{% include 'field.html.twig' with {
-  'icon': icon
-} %}
+{% embed 'field.html.twig' %}
+  {% block content %}
+    {% if multiple %}
+      <div class="field__items">
+    {% endif %}
+    {% for item in items %}
+      <div{{ item.attributes.addClass(item_classes) }}>
+        {{ icon }}
+        <time datetime="{{ item.content }}" class="datetime">{{ item.content }}</time>
```

Comment:
The datetime attribute isn't correctly formatted.

Reply:
See earlier comment about datetime and future refactoring.